ros/api/v1/hosts.py

```diff
@@ -206,10 +186,12 @@ def build_system_filters():
         if operating_systems := request.args.getlist('os'):
             modified_operating_systems = []
             for os in operating_systems:
-                os = os.split(" ")[1]
-                if os not in OS_VERSIONS.keys():
-                    abort(400, message='Not a valid RHEL version')
-                modified_operating_systems.append(OS_VERSIONS[os])
+                os_object = {"name": "RHEL"}
+                if len(os) == 1:
```

Comment:
@r14chandra what if I want all the systems with major version `8` and I don't care about the minor versions? With this if condition we will return the systems which are `8.0` specifically? and I think below two queries are different,
1. Get all RHEL 8 systems
2. Get RHEL 8.0 systems

What do you think?

Reply:
No, these would be treated as the same queries. Archives contain versions in `8.0` or `7.0` format not `8` or `7`.

Reply:
I was comparing the flow of UI with api and there frontend itself generates the major version filters upon getting minors and that confused me. So yeah, this looks good now from my side as well. Approving it!

Reply:
It can be addressed separately after discussing how we want to keep OS filtering via API going futher. 
Keeping this conversation open till our discussion.